assets/javascripts/discourse/components/chat-channel-settings-view.js

```diff
@@ -84,6 +99,17 @@ export default class ChatChannelSettingsView extends Component {
     );
   }
 
+  @computed(
+    "siteSettings.max_chat_auto_joined_users",
```

Comment:
siteSettings is a POJO it has no effect in computed, you can use it directly in the function